torch/csrc/utils/python_dispatch.cpp

```diff
@@ -772,6 +782,16 @@ void initDispatchBindings(PyObject* module) {
     return at::functionalization::impl::commit_update(a);
   });
 
+  m.def("_dispatch_key_for_device", [](const std::string& device_type) {
+    auto device = c10::Device(device_type);
+    TORCH_CHECK(
+        !device.has_index(),
+        "Expected device_type string to not a device index; got ",
```

Comment:
```suggestion
        "Expected device_type string to not have a device index; got ",
```

Reply:
also device_type -> device type? 
since we didn't actually name the argument to impl `device_type`